### PR TITLE
Fixed LUT truth table padding so that unconnected inputs are consider…

### PR DESCRIPTION
**Pull request by mkurc-ant**
This PR fixes LUT padding so that unused inputs can be left floating without influencing the output function.

### CODE REVIEW

**Comment by tpagarani** [comment]
@mkurc-ant, have you validated this change with simulation models?

**Comment by lnsharma** [comment]
@mkurc-ant : please see that you add testcase(s) to verify the issue fix. Now, Kishor has checked in the mechanism for adding testcases.

**Comment by mkurc-ant** [comment]
Yes, I've verified it on the counter design in simulation. I'll add test cases.

File: quicklogic/qlf_k4n8/utils/repacker/tests/lut_padding/lut1.eblif [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.model top
```
Comment (lnsharma) [inline quicklogic/qlf_k4n8/utils/repacker/tests/lut_padding/lut1.eblif:1]: Is this test folder part of github actions run? We need to see that these tests are run every time whenever a PR is created.

Comment (mkurc-ant) [reply quicklogic/qlf_k4n8/utils/repacker/tests/lut_padding/lut1.eblif:1]: Yes it is. I've connected the repacker tests to the `all_quicklogic_tests` target. See the changes to `quicklogic/qlf_k4n8/utils/CMakeLists.txt`.

Also when you browse logs from CI runs you should encounter lines like:
```
repacker/tests/lut_padding/test_lut_padding.py::test_lut_padding[1-1]
```
which indicated that the tests were run.

**Review by acomodi — APPROVED** [review]
LGTM

**Pull request merged**